lib/src/database/sqflite_box.dart

```diff
@@ -66,7 +66,7 @@ class BoxCollection with ZoneTransactionMixin {
         },
       );
 
-  Future<void> close() => _db.close();
+  Future<void> close() => zoneTransaction(() => _db.close());
```

Comment:
does the indexedDb impl not need this for some reason?

Reply:
I don't really know, it has an assertion, that it is not in a transaction...

Reply:
Ah, those were different transactions.

Reply:
ye but the assert basically throws when we close the db while in a txn, wouldn't it be better to close the db after txn was done? (so in a zonetxn?)